src/java.base/share/classes/jdk/internal/reflect/ReflectionFactory.java

```diff
@@ -638,10 +575,7 @@ private static boolean disableSerialConstructorChecks() {
     // for the first few invocations of Methods and Constructors and
     // then switch to the bytecode-based implementations.
 
-    private static final Config DEFAULT_CONFIG = new Config(false, // noInflation
-                                                            15, // inflationThreshold
-                                                            ALL_MH_ACCESSORS, // useDirectMethodHandle
-                                                            false, // useNativeAccessorOnly
+    private static final Config DEFAULT_CONFIG = new Config(false, // useNativeAccessorOnly
```

Comment:
The block comment just before this will need updating.  There's another one in Config.config where it uses the default/native implementation in early startup.